### ISSUE TITLE
NDS Mode Support 

### ISSUE THREAD

**Issue by Committee** [roxas75/rxTools#384]
Possible NDS Mode Support for next release?

**Comment by Zetchzie** [comment]
not a chance lol no one has added nds cartridge dump support to any cfw in existence

**Comment by urherenow** [comment]
@Zetchzie did I miss something? Who asked about nds cart dumping? I take this as a request (I'm assuming that it's targeted at n3ds) to support launching twl. Launching a cart (flash cart or not) doesnt require any patches at all (except for MAYBE the whitelist... But that's irrelevant to rxTools), so why does it not work? Even if it had to be launched from sysnand, it should be doable. You can launch an nds cart from reinand just fine.

**Comment by Zetchzie** [comment]
I mean support for loading nds roms dumped from cartridge on emunand. because this already supports dsiware. to an extent

**Comment by Committee** [comment]
NDS Mode is support for NDS Cartridges which currently do not work in RXTools and Emunand and yes I did not mention anything about NDS Catrdige dumping, I specifically said NDS MODE Support.

**Comment by urherenow** [comment]
That's what I thought. It may be a valid request, but I would think that it is extremely low priority, since anybody can simply chose to NOT load a cfw and play nds carts if they want to from a regular sysnand.

**Comment by Zetchzie** [comment]
Sorry for my misinterpretation.

**Comment by ApacheThunder** [comment]
Yeah, the ability to disable reboot patches in the config menu would be cool. It's just the reboot patches that break TWL booting from emunand. Reboot patches are needed for games like Smash Bros so it's understandable why it's on by default. n3DS users may not even need the reboot patches. Don't have an n3DS yet though so can't say for sure. But at least the ability to disable the reboot patches like CakesFW can would be cool. :P

**Comment by Committee** [comment]
Offtopic:

That avatar, if your the same person in GBATemp I would like you to know that your avatar always makes me stop scrolling and read your posts. Excellent Avatar.

**Comment by ApacheThunder** [comment]
lol. Thanks I guess. It's an actual photo of me. Was a joke photo I took on Facebook awhile back. Decided to crop it and make it my avatar. :P

**Comment by Zetchzie** [comment]
@Committee Lol same here ive seen ApacheThunder on various sites and he is quite knowledgeable so i read his stuff too.

**Comment by AlbertoSONIC** [comment]
We can't disable the reboothax patch, as it is used to patch the nativefirm too. I mean, not only when you reboot, but also when you boot the cfw!

**Comment by Committee** [comment]
Understood, good to close this then and thank you for the response.

**Comment by ApacheThunder** [comment]
I was referring to the "survive reboot patch" used for making sure  games like Super Smash Bros works on o3DS. CakesFW has it off by default and thus TWL works fine from emunand....I don't understand why rxTools is forced to use this patch....

**Comment by AlbertoSONIC** [comment]
@ApacheThunder Because the code that patches NATIVE FIRM is inside the reboot patch... Doesn't ask me why, it's @173210 idea. Btw we should add twl and agb support on n3ds emunand pretty soon so this won't be a problem

**Comment by 173210** [comment]
That's not right. reboot patch itself prevents from booting FIRMs in NAND whether it patches FIRM or not.
Anyway, it's possible to turn off the feature only for non-NATIVE_FIRM, but you don't have to do that if it will support TWL and AGB, do you? ;)
